core/src/main/scala/org/scalawag/jibe/mandate/GroupMandates.scala

```diff
@@ -1,14 +1,7 @@
 package org.scalawag.jibe.mandate
 
 import org.scalawag.jibe.backend.GroupResource
-
```

Comment:
I just moved this to the command package because, for the rest of the commands, their data classes live there.